install-tools.sh

```diff
@@ -0,0 +1,2 @@
+tools=$(go list -f '{{range .Imports}}{{.}} {{end}}' tools.go)
```

Comment:
This exists already, it's just in the hack directory

Reply:
Right! Removing...

Reply:
Removed